### ISSUE TITLE
script in Custom Queries?

### ISSUE THREAD

**Issue by alpglide** [prest/prest#784]
Custom Queries are really helpful.
When business logic is complex, it's not that easy to do in one sql.
is it possible to introduce executable scripts? for example, preset just provides http request parameters as json, run the script (the script can query prest multiple times if needed) to get output.
By this way,  users can use their preferred programming languages to implement business logic. 
script may not be that efficient, but it's the user's choice.

**Comment by alpglide** [comment]
I just realize there is one example at page https://docs.prestd.com/prestd/docs/api-reference/queries/ under split section:
 "SELECT * FROM table WHERE
name IN ({{ range $index,$part := split 'test1,test2,test3' `,` }}{{if gt $index 0 }},{{end}}'{{$part}}'{{ end }});"
any detail about if, range? thanks

**Comment by vmesel** [comment]
Hey @alpglide! They come as quotes because URL fields are strings whenever you pull them.

One way you can use this data is casting it, which is costly, but depending on the table size, it’s a worthy try.

**Comment by alpglide** [comment]
> Hey @alpglide! They come as quotes because URL fields are strings whenever you pull them.
> 
> One way you can use this data is casting it, which is costly, but depending on the table size, it’s a worthy try.
yes, it works. your help is appreciated!

**Comment by alpglide** [comment]
> 

Yes, that's my preferred way. but other team members do not want to do so. They believe it's hard to understand and maintenance. :)
